client/package.json

```
@@ -32,7 +32,7 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "lint": "eslint '*/**/*.{js,ts,tsx}' --fix"
+    "lint": "eslint '*/**/*.{js,ts,tsx}' --fix --no-error-on-unmatched-pattern"
```

--no-error-on-unmatched-pattern might hide the fact that no files were matched due to syntax errors. Was a backslash added to the double-quote when you tried that earlier?
```suggestion
    "lint": "eslint \"*/**/*.{js,ts,tsx}\" --fix"
```

---

Yeah backslash was. I'll try again later.

---

No worries, just making sure every possibility is exhausted.